devnet/src/mpc.rs

```diff
@@ -909,3 +910,141 @@ struct VoteNewParametersArgs {
     prospective_epoch_id: EpochId,
     proposal: ThresholdParameters,
 }
+
+impl MpcDescribeCmd {
+    pub async fn run(&self, name: &str, config: ParsedConfig) {
```

Comment:
Wdyt of adding an http endpoint to each mpc node which prints the current contract view? It could use the same function as this one and be used by opertators to ensure they vote for the right things.

Reply:
I love that idea. I do feel that a lot of functionality here is also useful for #344 .